docs/tutorial/06-fungible-tokens.md

```diff
@@ -832,7 +859,7 @@ transaction {
 		let vaultA <- ExampleToken.createEmptyVault()
 
 		// Store the vault in the account storage
-		acct.storage.save(<-vaultA, to: /storage/CadenceFungibleTokenTutorialVault)
+		acct.save<@ExampleToken.Vault>(<-vaultA, to: /storage/CadenceFungibleTokenTutorialVault)
```

Comment:
is this meant to be `acct.storage.save`?

Reply:
this whole tutorial still hasn't been updated to C1.0 yet, so it is all using the old version here anyway